recipes/azure-mgmt-subscription/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "azure-mgmt-subscription" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: 37f570b8872ae65dce312da116588ab8407a5c8a10d959597e61d19b21799f77
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - msrest >=0.5.0
+    - msrestazure >=0.4.32,<2.0.0
+    - azure-common ~=1.1
+
+test:
+  imports:
+    - azure
+    - azure.mgmt
+    - azure.mgmt.subscription
```

Can we try to add ``pip check`` here please?
```suggestion
    - azure.mgmt.subscription
  requires:
    - pip
  commands:
    - pip check
```